mmv1/third_party/terraform/utils/gcp_sweeper_test.go

```diff
@@ -1,9 +1,11 @@
-package google
+package google_test
```

Comment:
```suggestion
package google
```

google_test doesn't seem correct? I've never seen this pattern before. The `_test` on the file name is how go makes the distinction.

Reply:
package_test should also work. Please see https://pkg.go.dev/testing

Here I changed the package to `google_test` for testing. I will change it back to `google`

Reply:
Ah, so this is to keep the namespaces distinct!